modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/getRandomId.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export default function getRandomId() {
+	return Math.random().toString(36).substring(2, 9);
+}
```

Comment:
Along with adding new util, could we remove [the existing one](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/index.js#L73-L75) and its one usage?